Client/src/Components/Inputs/Checkbox/index.jsx

```diff
@@ -36,10 +38,12 @@ const Checkbox = ({
 	value,
 	onChange,
 	isDisabled,
+	alignSelf
 }) => {
 	/* TODO move sizes to theme */
 	const sizes = { small: "14px", medium: "16px", large: "18px" };
 	const theme = useTheme();
+	const override = alignSelf?  { alignSelf: "flex-start" } : {}
```

Comment:
I am not sure what this is doing, can you provide a print screen @shemy-gan ?

Reply:
The use case is when the label is an element instead of string, it will align the checkbox on top of the container vertically; Alex suggested to expose this explicitly, so the fix was here :https://github.com/bluewave-labs/checkmate/pull/1306/commits/54c2826205c3aaa5413eb0394af044457c8b5fc4

Reply:
Can you show the two cases in a print screen?

Reply:
Please move on to feat/fe/statuspage 2, the layout is as below
![Screenshot from 2024-12-11 19-28-38](https://github.com/user-attachments/assets/f685b094-4f9e-428b-b55c-71d0d1860b55)

Reply:
"Please move on to feat/fe/statuspage 2" what do you mean?

Reply:
Oh, ok. So should we close this PR, since it is already contemplated in the other one?